docs/installation.rst

```diff
@@ -86,7 +86,7 @@ Installing the Required Data Files
 
 Files containing such information as the JWST pupil shape, instrument throughputs, and aperture positions are distributed separately from WebbPSF. To run WebbPSF, you must download these files and tell WebbPSF where to find them using the ``WEBBPSF_PATH`` environment variable.
 
-1. Download the following file:  `webbpsf-data-0.7.0.tar.gz <http://www.stsci.edu/~mperrin/software/webbpsf/webbpsf-data-0.7.0.tar.gz>`_  [approx. 240 MB]
+1. Download the following file:  `webbpsf-data-0.7.0.tar.gz <http://www.stsci.edu/~spacetelescope/software/webbpsf/webbpsf-data-0.7.0.tar.gz>`_  [approx. 240 MB]
```

Comment:
This one we shouldn’t change - that URL won’t work to download the zip file from ~spacetelescope since there’s no such user.